packages/wonder-blocks-accordion/src/components/accordion.tsx

```diff
@@ -162,56 +161,63 @@ const Accordion = React.forwardRef(function Accordion(
         newSectionsOpened[index] = newOpenedValueAtIndex;
         setSectionsOpened(newSectionsOpened);
 
-        // Keep track of the currently focused section for keyboard navigation.
-        setCurrentlyFocusedSection(index);
-
         if (childOnToggle) {
             childOnToggle(newOpenedValueAtIndex);
         }
     };
 
-    const handleSectionFocus = (index: number) => {
-        setCurrentlyFocusedSection(index);
-    };
-
-    /** Keyboard navigation for keys: ArrowUp, ArrowDown, Home, and End.
-     *
-     * Note that we don't have to use `setCurrentlyFocusedSection` in this
-     * function because the focus is handled by the browser + the
-     * section's onFocus handler (the `handleSectionFocus` function above).
+    /**
+     * Keyboard navigation for keys: ArrowUp, ArrowDown, Home, and End.
      */
     const handleKeyDown = (event: React.KeyboardEvent) => {
+        // From https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
```

Comment:
**praise:** This is great! really helps to future selves to understand why we are doing it this way.